src/shrutiDJun20/SumIs7.java

```diff
@@ -0,0 +1,36 @@
+/*Find all pairs on integer array whose sum is equal to given number?
+Hint: You have given an integer array and a number, you need to write a program to find
+all elements in the array whose sum is equal to the given number. Remember, the array
+may contain both positive and negative numbers, so your solution should consider that.
+Input:2, 4, 3, 5, 7, 8, 9
+Given sum:7
+Integer numbers, whose sum is equal to value:7
+(2, 5)
+(4, 3)
+ */
+
+package shrutiDJun20;
+
+public class SumIs7 {
+	
+	void sumEqualsSeven(int[] arr){
+		//int sum = 0;
+		System.out.println("Integer numbers, whose sum is equal to value:7");
+		for(int index=0;  index<arr.length; index++)
+			
+			for(int index2=0;  index2<arr.length; index2++){
+				if(arr[index]+arr[index2]==7){
```

Comment:
This is correct but to safer side always do () for mathematics operation in java

Reply:
Ok Harsh.. will keep in mind.. thank you.